package.json

```diff
@@ -2,22 +2,6 @@
   "name": "klombies-games-frontend",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/styles": "^4.10.0",
```

Comment:
The `@material-ui/styles` package is bundled with `core` so you don't need the explicit dependency here.
Reference: https://material-ui.com/styles/basics/#installation